src/components/interpolatedColorLegend.ts

```diff
@@ -197,7 +198,7 @@ module Plottable.Components {
       let text1 = this._formatter(domain[1]);
       let text1Width = this._measurer.measure(text1).width;
 
-      let textHeight = this._measurer.measure().height;
+      let textHeight = this._measurer.measure(text0).height;
```

Comment:
We should probably just use `this.measurer.measure().height`, which uses the tall text "bqpdl".